experiments/ssl4eo/download_l5_raw.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash -euo pipefail
+
+# User-specific parameters
```

Comment:
These are parameters that people will need to change depending on where and how they will run the script.